apps/dash-image-segmentation/app.py

```diff
@@ -329,21 +336,37 @@ def show_segmentation(image_path, mask_shapes, segmenter_args):
     segimg, _, clf = compute_segmentations(
         mask_shapes,
         img_path=image_path,
-        segmenter_args=segmenter_args,
         shape_layers=shape_layers,
         label_to_colors_args=label_to_colors_args,
+        features=features,
     )
     # get the classifier that we can later store in the Store
-    classifier = save_img_classifier(clf, segmenter_args, label_to_colors_args)
+    classifier = save_img_classifier(clf, label_to_colors_args)
```

Comment:
I need to find a way to put back the `segmenter_args`, since it's needed to compute features but I thought I'd submit first an early version of this PR to your review.